mscore/exportxml.cpp

```diff
@@ -277,12 +277,13 @@ class ExportMusicXml {
       Ottava const* ottavas[MAX_NUMBER_LEVEL];
       Trill const* trills[MAX_NUMBER_LEVEL];
       int div;
-      double millimeters;
-      int tenths;
+      double scale;
```

Comment:
fyi - This module has been refactored in a major way.  The page settings numeric formulas are completely redesigned and greatly simplified.